packages/admin/admin-theme/src/componentsTheme/MuiMenu.tsx

```diff
@@ -0,0 +1,23 @@
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiMenu: GetMuiComponentTheme<"MuiMenu"> = (component, theme) => ({
+    ...component,
+    defaultProps: {
+        ...component?.defaultProps,
+    },
+    styleOverrides: mergeOverrideStyles<"MuiMenu">(component?.styleOverrides, {
+        paper: { minWidth: 220, borderRadius: "4px" },
+        root: {
+            "& .MuiMenuItem-root": {
+                padding: "8px 15px 8px 30px !important",
```

Comment:
I don't think you need the `!important` here 🤔 

```suggestion
                padding: "8px 15px 8px 30px",
```